docs/docs/integrations/mixpanel.md

```diff
@@ -72,3 +72,39 @@ Every time an `Identity` requests their flags from the Flagsmith API, Flagsmith
  }
 }
 ```
+
+## Getting Mixpanel Cohorts into Flagsmith
```

Comment:
Ok, I think I understand now. So the idea is: 

Identity cohort changes in Mixpanel -> Triggers Webhook -> Hits endpoint on customer infrastructure -> Customer triggers request to Flagsmith to set traits. 

I don't think we've mentioned here anywhere that there will need to be that 'middleman' service in the client's infrastructure. Perhaps it's obvious but I think it's best to be explicit.  

We should probably make also sure people are aware that they can remove traits by setting the `trait_value` as `null`